internal/controller/maintenancemode_controller.go

```diff
@@ -55,11 +55,6 @@ func (r *MaintenanceModeReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			&handler.EnqueueRequestForObject{},
 			builder.WithPredicates(generationChangePredicate),
 		).
-		Watches(
-			&v1alpha1.StorageClaim{},
-			&handler.EnqueueRequestForObject{},
-			builder.WithPredicates(generationChangePredicate),
-		).
```

Comment:
You are removing the watch but you don't address the code in the controller that deals with StorageClaim resources